config/dev.exs

```diff
@@ -36,5 +36,5 @@ config :course_planner,
   auth_email_from_email: "dev@email"
 
 config :recaptcha,
-  public_key: "6LeVEDUUAAAAABUMP9L69uxFXKVlRHPkEqMH4_SG",
-  secret: "6LeVEDUUAAAAABevo88bjb4_yZCI7R2LZizynw2z"
+  secret: "${RECAPTCHA_SECRET_KEY}",
+  public_key: "${RECAPTCHA_SITE_KEY}"
```

Comment:
Please also include a `.envrc.sample` with the test keys so we can set it up in dev.